externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala

```diff
@@ -54,8 +54,12 @@ class SparkSQLSessionManager private (name: String, spark: SparkSession)
     try {
       val sparkSession = spark.newSession()
       conf.foreach {
-        case (HIVE_VAR_PREFIX(key), value) => setModifiableConfig(sparkSession, key, value)
-        case (HIVE_CONF_PREFIX(key), value) => setModifiableConfig(sparkSession, key, value)
+        case (HIVE_VAR_PREFIX(key), value) if key.startsWith(SPARK_PREFIX) =>
```

Comment:
can we remove HIVE_VAR_PREFIX and HIVE_VAR_PREFIX, just use `if else` like KyuubiSessionImpl?

Reply:
I found some negative cases that these regexes that didn't match well, so I switch to match case by case

Reply:
OK, I will change it